src/dc/components/venue-context/theme-context.js

```diff
@@ -11,10 +11,9 @@ const getDate = (strDate, defaultValue) =>{
 }
 
 export const venueConfig={
-    originIata:venue.originIata,
-    destinationIata:venue.destinationIata,
-    originLocation:venue.originLocation,
-    destinationLocation:venue.destinationLocation,
```

Comment:
Please keep a reference to the exact venue within the city, this is **Agora Bogotá Convention Center.**

Reply:
venueConfig usage is slightly changed. for the navigation purposes, I have extended this configuration.
Now not only IATA code is configured for the origin and destination but the whole airport object.
```
originAirport: venue.originAirport,
destinationAirport: venue.destinationAirport,
destinationLocation: venue.destinationLocation,
venueName: venue.venueName
```